modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/assignee/AssigneeDDMFormFieldTemplateContextContributor.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.dynamic.data.mapping.form.field.type.internal.assignee;
+
+import com.liferay.dynamic.data.mapping.form.field.type.DDMFormFieldTemplateContextContributor;
+import com.liferay.dynamic.data.mapping.model.DDMFormField;
+import com.liferay.dynamic.data.mapping.render.DDMFormFieldRenderingContext;
+import com.liferay.object.dynamic.data.mapping.form.field.type.constants.ObjectDDMFormFieldTypeConstants;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Carolina Barbosa
+ */
+@Component(
+	property = "ddm.form.field.type.name=" + ObjectDDMFormFieldTypeConstants.ASSIGNEE,
+	service = DDMFormFieldTemplateContextContributor.class
+)
+public class AssigneeDDMFormFieldTemplateContextContributor
+	implements DDMFormFieldTemplateContextContributor {
+
+	@Override
+	public Map<String, Object> getParameters(
+		DDMFormField ddmFormField,
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
+
+		return HashMapBuilder.<String, Object>put(
+			"value",
+			() -> {
+				String value = ddmFormFieldRenderingContext.getValue();
+
+				if (Validator.isNull(value)) {
```

Comment:
Hi @carolmariaabb, I use the `GridDDMFormFieldTemplateContextContributor` exemple to validate the value. Could you double check?